src/main/java/io/confluent/examples/streams/microservices/FraudService.java

```diff
@@ -100,19 +105,21 @@ private KafkaStreams processStreams(final String bootstrapServers, final String
     //as caching in Kafka Streams will conflate subsequent updates for the same key. Disabling caching ensures
     //we get a complete "changelog" from the aggregate(...) step above (i.e. every input event will have a
     //corresponding output event.
-    final Properties props = baseStreamsConfig(bootstrapServers, stateDir, SERVICE_APP_ID);
-    props.setProperty(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, "0");
+    final Properties config = baseStreamsConfig(bootstrapServers, stateDir, SERVICE_APP_ID);
+    config.setProperty(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, "0");
+    config.put(SCHEMA_REGISTRY_URL_CONFIG, schemaRegistryUrl);
```

Comment:
This change is not expected. For this example, all Serdes are manually configured, and we keep this code because the `Serdes` are also used for regular producers/consumers to write input data and read results.

However, even if all Serdes are manually configured, we still need to set the SR config here to not break the code -- without it, the second call to `configure()` remove the SR config and we fail with an exception at runtime, because the Serdes are not configures correctly and cannot connect to the SR. In fact, it fails with a NPE (cf. https://github.com/confluentinc/schema-registry/issues/1085)